miwa.rb

```diff
@@ -0,0 +1,45 @@
+class Indicatory
+    puts 'Щосаме вам потрібно порахувати ?'
+    answer = gets.chomp.upcase 
+    if answer == 'Electric'
+      @electric = gets.chomp 
+    elsif answer == 'Gas'
+      @gas = gets.chomp
+    elsif answer = 'Water'
+      @coldWater = gets.chomp
+      @hotWater = gets.chomp
+    else 
+        puts 'Виберіть одну з '
+    end 
+end
+
+class Electric
+
+    def electric
```

Comment:
It should be named more commonly, the method returns the price of the product, so it should be something like `payment` `price` etc.